src/LaravelCommon.php

```diff
@@ -2,12 +2,32 @@
 
 namespace LiveIntent\LaravelCommon;
 
+use Illuminate\Auth\RequestGuard;
+use Illuminate\Support\Facades\Auth;
 use Illuminate\Support\Facades\Queue;
 use Illuminate\Support\Facades\Route;
 use Illuminate\Foundation\Http\Events\RequestHandled;
+use LiveIntent\LaravelCommon\Auth\Guards\LITokenGuard;
 
 class LaravelCommon
 {
+    /**
+     * Register Tessellate token authentication guard
+     */
+    public static function authGuard()
+    {
+        Auth::extend('li_token', function ($app, $_name, array $config) {
+            throw_if($config['provider'] == null);
+
+            return new RequestGuard(function ($request) use ($config) {
+                return (new LITokenGuard(
+                    Auth::createUserProvider($config['provider']),
+                    config('auth.li_token.keys.public') ?? config('liveintent.auth.li_token.keys.public')
```

Comment:
Not sure why do this, can we make sure it is always either `auth.li_token.keys.public` or always `liveintent.auth.li_token.keys.public`

Reply:
I do agree with @jiechaowang here, I would stick to one approach, and probably I'd prefer to keep the current approach of the liveintent file.

Side note, the config function allows a second parameter for a default value. Just another reason why I feel like this looks a bit "off" to me.

Reply:
Yeah, I was on the fence on whether or not I wanted to do this--originally I was thinking if there was a reason to override it, but keep the liveintent config file untouched. However, I think you guys are right that it should just work one way.